datafusion/core/src/catalog/information_schema.rs

```diff
@@ -141,6 +154,12 @@ impl InformationSchemaProvider {
                 COLUMNS,
                 TableType::View,
             );
+            builder.add_table(
+                &catalog_name,
+                INFORMATION_SCHEMA,
+                SETTINGS,
+                TableType::View,
+            );
```

Comment:
add `SETTINGS` into `information_schema`

so that

```bash
❯ show tables;
+---------------+--------------------+------------+------------+
| table_catalog | table_schema       | table_name | table_type |
+---------------+--------------------+------------+------------+
| datafusion    | information_schema | tables     | VIEW       |
| datafusion    | information_schema | views      | VIEW       |
| datafusion    | information_schema | columns    | VIEW       |
| datafusion    | information_schema | settings   | VIEW       |
+---------------+--------------------+------------+------------+
4 rows in set. Query took 0.004 seconds.
```